test/selenium/src/tests/test_snapshots.py

```diff
@@ -167,6 +166,8 @@ def test_audit_contains_snapshotable_control_after_deleting_original_control(
                        get_list_objs_from_tree_view(src_obj=audit))
     assert [expected_control] == actual_controls, (
         messages.ERR_MSG_FORMAT.format([expected_control], actual_controls))
+    (webui_service.ControlsService(selenium).
```

Comment:
This test should be separated or parametrized into 2 tests:
1. delete control and do not update to latest version
2. delete control and update to latest version

I both cases asserts should go after required actions.